openslides_backend/action/mediafile/update.py

```diff
@@ -0,0 +1,58 @@
+from typing import Any, Dict
+
+from ...models.models import Mediafile
+from ...shared.patterns import Collection, FullQualifiedId
+from ..default_schema import DefaultSchema
+from ..generics import UpdateAction
+from ..register import register_action
+
+
+@register_action("mediafile.update")
+class MediafileUpdate(UpdateAction):
+    """
+    Action to update a mediafile.
+    """
+
+    model = Mediafile()
+    schema = DefaultSchema(Mediafile()).get_update_schema(
+        optional_properties=["title", "access_group_ids"]
+    )
+
+    def update_instance(self, instance: Dict[str, Any]) -> Dict[str, Any]:
```

Comment:
You have to also update all children if either `inherited_access_group_ids` or `has_inherited_access_group_ids` change since their respective `inherited_access_group_ids` may change then, too.

Reply:
How do I change more then one instance with the update_instance method? I think, this is not possible.